### DISCUSSION TITLE
Pasting from another note created a table?

### DISCUSSION THREAD

**Discussion by LorenAmelang** [TriliumNext/Trilium#2617] [Q&A]
I had two notes clipped from the web, neither showing any sign of including tables. I wanted to combine them, so I copied the content of one and pasted it to the other. The pasted text had a line all around the outside, which I finally figured out was a 1x1 table. It had the table attributes like this: 
![table](https://user-images.githubusercontent.com/1297090/152734718-2786462b-3670-4568-8857-303a5b18d6e9.PNG)
Why did ordinary text become a table? 

I see how to modify a multi-cell table, but it seems 1x1 is the lowest you can go. Is there some way to remove the table surround and just leave the text, like it was in the original note? 

Is this feature, and the buttons I had to just blindly try out, documented somewhere?

**Comment by zadam** [comment]
Could you check the markup of the original note to make sure there was no table?

![image](https://user-images.githubusercontent.com/617641/152878353-758a61a4-4154-43a5-b1b4-885ec736fcf6.png)

Clipped pages often have weird things inside, perhaps it was not visible because it was readonly?

**Reply by LorenAmelang** [reply]
I had already deleted the two files and started over, manually copying plain text. 
I just clipped them again and there is no occurrence of "<t" in either one. 

(Is there a search within Note source? Ctrl+F draws its box but is not accessible from there. I copied them to an external editor.) 

But the old clips were from months ago, so who knows...  I just tried the paste again with the new clips and did not get the table outline. They are pages from a forum on https://groups.io, lots of dividers and quotes and photos, but nothing I'd visually recognize as a table. 

I'm wondering if maybe I clicked the table button in the copied note sometime in the past, not knowing what it was. Seems like they should all have popup labels when you mouse over them...  But no table outline was visible on the source note. If it had a one pixel table hiding somewhere, could pasting it move the entire content into a table? 

No, just tried that, got a one line by full note width outline, which pasted into the other note exactly as it appeared in the first. And could be selected and deleted. The problem outline looked like the table outline, had the same icon above top left, but contained all of the copied text. 

At least now I know where to find the source if it happens again! Thanks!

**Reply by LorenAmelang** [reply]
Deeper in the rabbit hole...

Just realized I could Undelete the old notes. There were three in the list. But Undeleting the oldest one removed the Undelete option from all three. Only one name re-appeared in the tree. And it can't be opened. Shows a right click menu, but left click just opens the view of contained image notes. 

Maybe the whole problem was that Clipper gave them the same note name? They were multiple web page sections of the same forum post, same title. And I didn't rename them manually. Does the visible note name matter? Didn't seem to be a problem in today's copy/paste test...

**Reply by zadam** [reply]
There seems to be some problem in the frontend after undelete. Restart/reload should help.

Having same note title is not a problem in Trilium.

**Reply by LorenAmelang** [reply]
"Reload frontend" allowed yesterday's undeleted item in the tree to open - but it shows the current state of the note I re-created and renamed, not any of the previous deleted "table" versions. 

It also made all the history links look live again, but clicking any of the three of them opens the same note I undeleted yesterday - even though I've since renamed it to "reload ...". 

Supposedly undeleted:
Note ID: | p2AcONjxAA82 | Created: | 2022-02-01 14:11 | Modified: | 2022-02-08 17:08
text (text/html) | 136 KiB (subtree size: 193 KiB in 4 notes)
(Guess undelete is a modification?)
(Created timestamp is when I originally clipped both of the copy/paste notes.)

I deleted both of them before I re-clipped the two below:

Current:
Note ID: | P7Pw028UUQD2 | Created: | 2022-02-07 14:11 | Modified: | 2022-02-07 15:02
text (text/html) | 183 KiB (subtree size: 240 KiB in 4 notes)

Paste target for Current:
Note ID: | QbjsT1DyrzXu | Created: | 2022-02-07 14:08 | Modified: | 2022-02-07 15:02
text (text/html) | 42 KiB (subtree size: 70 KiB in 3 notes)

Sure looks like it somehow updated that new content into the already deleted old notes with the same clipped names, even though I renamed the new clips before copy/pasting them. Clearly they have unique IDs...  

(This is not a huge problem for my actual note taking, I'm just captivated by things I don't understand.)
